assets/js/blocks/products/utils.js

```diff
@@ -35,7 +35,7 @@ export const renderNoProductsPlaceholder = ( blockTitle, blockIcon ) => (
 			className="wc-block-products__add-product-button"
 			isDefault
 			isLarge
-			href={ adminUrl + 'post-new.php?post_type=product' }
+			href={ ADMIN_URL + 'post-new.php?post_type=product' }
```

Comment:
wowzer! good catch here!

Reply:
this was from another PR, not sure how it got here 😅, props to Albert.

Reply:
this was from another PR, not sure how it got here 😅, props to Albert.